benchmarking/utils/parse_bool.py

```diff
@@ -0,0 +1,10 @@
+
+
+def parse_bool(val: str) -> bool:
```

Comment:
Same thing here, please import from benchmarking/utils/__init__.py. Feel free to put these different functions into a single file, and import that one in __init__.py. They don't really warrant different files.